### ISSUE TITLE
'impact.game' is not actually required by Weltmeister

### ISSUE THREAD

**Issue by Joncom** [phoboslab/Impact#64]
https://github.com/phoboslab/Impact/blob/70f34c803acc39ff1ec2b706a5be512dc4275e0d/lib/weltmeister/weltmeister.js#L7-L9

`ig.Game` is not used, so this line doesn't need to be here, right?

**Comment by phoboslab** [comment]
Right. I think this is in there as a shorthand to require `impact.entity`.: https://github.com/phoboslab/Impact/blob/70f34c803acc39ff1ec2b706a5be512dc4275e0d/lib/impact/game.js#L4-L9
